apps/nextjs-website/react-components/types/NotFoundPage/NotFoundPage.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactElement } from 'react';
+import { SectionProps } from '@react-components/types/common/Common.types';
+
+export interface NotFoundPageProps extends SectionProps {
+  image: ReactElement;
+  mobileImage: ReactElement;
+  redirectUrl?: string;
```

Comment:
```suggestion
  redirectUrl: string;
```
Let's make this required to ensure we don't forget to localize the redirectUrl when using the component.